test/Mvc6Framework45.FunctionalTests/Startup.cs

```diff
@@ -124,8 +123,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             {
                 routes.MapRoute(
                     name: "default",
-                    template: "{controller}/{action}/{id?}",
-                    defaults: new { controller = "Home", action = "Index" });
+                    template: "{controller=Home}/{actionIndex}/{id?}");
```

Comment:
BTW an error like this would lead me to believe that this test doesn't run, because there's no way this would work. MVC actions wouldn't be dispatched. Can you double check that all the tests are running as expected?